### DISCUSSION TITLE
Implement one-to-many attributes ER using Splink?

### DISCUSSION THREAD

**Discussion by arunism** [moj-analytical-services/splink#997] [Q&A]
I have a database where i need to match its `names` column with `multiple columns` (like brand name, national name, company name, and so on) of another database. We are not sure which column from the second database matches to the name column from the first database. So we need to compare one column with all from the another one. I was curious if there is any way to implement one-to-many attributes Entity Resolution using Splink?

**Comment by RobinL** [comment]
Yes.  Sorry - don't have time right now to provide a detailed example but if you read this:
https://moj-analytical-services.github.io/splink/topic_guides/customising_comparisons.html#method-3-providing-the-spec-as-a-dictionary

you'd want a `sql_condition` something like `"sql_condition": "name_l = brand_name_r OR name_l = national_name_r etc.",`
